src/Sylius/Bundle/ChannelBundle/Resources/views/FakeChannelCollector/layout.html.twig

```diff
@@ -4,8 +4,8 @@
     {% set icon %}
         <a href="#">
             {# fake image span #}<span style="width: 0; height: 28px; vertical-align: middle;"></span>
-            <span class="sf-toolbar-status">{{ collector.channels|length }}</span> channels,
-            <b>{{ collector.currentChannel.name }}</b>
+            <span class="sf-toolbar-status">{{ collector.channels|length }}</span> channel(s),
+            <b>{{ collector.currentChannel.name|default('Undefined') }}</b>
```

Comment:
Anyway, we may save only `current_channel_name` instead of `current_channel` in WDT.